src/ui/services/products/add-new-product.ui-service.ts

```diff
@@ -0,0 +1,33 @@
+import { expect, Page } from "@playwright/test";
+import { MY_USER } from "config/envirement";
+import { HomePage } from "ui/pages/home.page";
+import { SignIn } from "ui/pages/login/sign-in.page";
+import { ProductsPage } from "ui/pages/products/products.page";
+import { AddNewProductPage } from "ui/pages/products/add-new-product.page";
+import { generateCustomerData } from "data/products/generateProduct.data";
+import { apiConfig } from "config/api-config";
+import { IProductResponse } from "types/product.types";
+import { STATUS_CODES } from "data/statusCodes";
+import _ from "lodash";
+
+export class AddNewProductUIService {
+  private productsPage: ProductsPage;
+  private addNewProductPage: AddNewProductPage;
+  constructor(private page: Page) {
+    this.productsPage = new ProductsPage(page);
+    this.addNewProductPage = new AddNewProductPage(page);
+  }
```

Comment:
Можно добавить класс `PageHolder` от которого будут наследоваться все кому нужна `Page`

```suggestion

export class PageHolder {
  constructor(protected page: Page) {}
}
export class AddNewProductUIService extend  PageHolder {
    private productsPage: ProductsPage = new ProductsPage(this.page);
    private addNewProductPage: AddNewProductPage = new AddNewProductPage(this.page);
  }
```